### PR TITLE
Added hover feature.

### PR DESCRIPTION
**Pull request by Visham32**
When the user hover on a specific tab, the tab will move by 4px on the right as enhancement. Also the pointer changes to a hand cursor when hovering. However, i tried changing the background color but the effect is not happening.

### CODE REVIEW

File: app/styles/main.css [hunk 1]
```diff
@@ -205,3 +205,10 @@ label {
   text-overflow: ellipsis;
   margin-bottom: 20px;
 }
+
+a:hover {
+  color: #000000;
+  background-color:#ff6363;  
+  margin-right:4px; /*moves the specific tab on the right by 4px when hover*/
```
Comment (jaylenw) [inline app/styles/main.css:212]: I am going to remove this hover on right functionality. It doesnt work well when buttons are next to each other.

File: app/styles/main.css [hunk 2]
```diff
@@ -205,3 +205,10 @@ label {
   text-overflow: ellipsis;
   margin-bottom: 20px;
 }
+
+a:hover {
+  color: #000000;
+  background-color:#ff6363;
```
Comment (jaylenw) [inline app/styles/main.css:211]: i do like this color

**Review by jaylenw — COMMENTED** [review]
Thanks for the submission, I am just going to change a little bit and merge this in.

**Comment by Visham32** [comment]
Thank you for the merge.

**Pull request merged**